MovieGPT-api/src/main/java/lt/techin/group/project/security/SecurityConfig.java

```diff
@@ -0,0 +1,55 @@
+package lt.techin.group.project.security;
+
+import lombok.AllArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@AllArgsConstructor
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
+
+    private JwtAuthenticationFilter jwtAuthenticationFilter;
+    private UserDetailsServiceImpl userDetailsService;
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http
+                .csrf(csrf -> csrf.disable())
```

Comment:
This can be  .csrf(AbstractHttpConfigurer::disable)

Reply:
We still need cors so developers on react can access our endpoints
.cors(Customizer.withDefaults())
I used this in previous projects

Reply:
Cors added and test, now endpoints have access from http://localhost:5173